rating-service/src/main/java/edu/miu/ratingservice/dto/request/RatingRequestDTO.java

```diff
@@ -0,0 +1,12 @@
+package edu.miu.ratingservice.dto.request;
+
+import java.io.Serializable;
+
+import lombok.Data;
+
+@Data
+public class RatingRequestDTO implements Serializable {
+    private String dailyMeal;
```

Comment:
Since it's the id, it's better to call it dailyMealId.
Same thing for the user, and same thing for those fields in the RatingResponseDTO.